mods/parsoid.js

```diff
@@ -459,7 +460,27 @@ PSP.transformRevision = function (restbase, req, from, to) {
         var body2 = {
             original: original
         };
-        body2[from] = req.body[from];
+        if (from === 'sections') {
+            try {
+                var sections = JSON.parse(req.body.sections);
```

Comment:
Ah, ok, I'll add JSON handling. 

Regarding your second comment, it would be not `'sections=1234'.toString()` but `1234.toString()` but anyway this is not relevant now. I'll reupload the PR after I work out son handling.